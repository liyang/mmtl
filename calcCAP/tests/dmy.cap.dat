57.0
40.0
12.2807017544
7.01754385965
-9.5
0
1024
40
6
8
10
5
0.0
1.0e5
2
5
3.0siemens/meter
2 2 0 0 0 0 1
DIELE 2 recTags1 1 0  1e-05 57.0  20 4.26 DIELE 5 recTags1 1 0  20.00001 57.0  20 4.26 
REC 3 recTags1 1 19.0  20.00001 7  3 REC 4 recTags1 1 31.0  20.00001 7  3 




GND 1 gndTags1 1 0 0 57.0 10 0
