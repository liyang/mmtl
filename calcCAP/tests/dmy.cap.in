1
2
0
0
2
0
0
57.0
10
19.0
20.00001
7
3
31.0
20.00001
7
3
0
1e-05
57.0
20
4.26
0
20.00001
57.0
20
4.26
1024
40
6
8
10
5
0.0
1.0e5
2
5
3.0siemens/meter
