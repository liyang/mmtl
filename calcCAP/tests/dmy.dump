

		DUMP OF GEOMETRY

cntr_seg=10, pln_seg=10, coupling=0.0254, risetime=1e-09
conductivity=5.8e+07, frequency=1e+08, half min dim=1.5
1 ground planes, top thickness=1, bottom thickness=1

SIGNAL CONDUCTORS:

R Rect2R1 rectangle
	low left = (12,20), up right = (19,23)

R Rect2R0 rectangle
	low left = (0,20), up right = (7,23)


GROUND CONDUCTORS:


DIELECTRICS:

	low left = (-19,20), up right = (38,40)
	constant=4.26, tangent=0

	low left = (-19,0), up right = (38,20)
	constant=4.26, tangent=0


