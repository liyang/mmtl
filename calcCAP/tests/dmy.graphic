$ICON;
$DESIGN "";
$ORIGIN (4.16,0);
$PINGRID (0,0);
		$IATT;
		"EM_TYPE" (0,0) "tran_line" 2,0.06,1,2,1,1,0,0;
		"PAGE1" (0,0) "10001" 2,0.06,1,2,1,1,0,0;
		$END;
$PINS;
$END;
$PRIMITIVES;
$END;
$CONNECTS;
$END;
$TEXT;
$END;
$OBJECT;
$PAGE 1;
$PAGESIZE (11,8.5);
$DESIGN "_1";
$ORIGIN (0,0);
$PINGRID (0,0);
		$IATT;
		"COUPLING_LENGTH" (0,0) "0.0 meters" 2,0.06,1,2,1,1,0,0;
		"RISETIME" (0,0) "0.0 ps" 2,0.06,1,2,1,1,0,0;
		"CONTOUR_SEGMENTS" (0,0) "10" 2,0.06,1,2,1,1,0,0;
		"PLANE_SEGMENTS" (0,0) "10" 2,0.06,1,2,1,1,0,0;
		$END;
$PINS;
$END;
$PRIMITIVES;
	I ground_plane (0,0)(0,0)0,0,0;
		$ATT;
		"COMP" (0,0) "10001" 2,0.06,1,2,1,1,0,0;
		$END;
	R(0,0.32)(11,3.07)2,0,0;
		$ATT;
		"LOWER_LEFT" (0,0) "( -30.5 meters , 0 meters )" 2,0.06,1,2,1,1,0,0;
		"UPPER_RIGHT" (0,0) "( 49.5 meters , 20 meters )" 2,0.06,1,2,1,1,0,0;
		"RELATIVE_PERMITTIVITY" (0,0) "4.26" 2,0.06,1,2,1,1,0,0;
		"LOSS_TANGENT" (0,0) "0.014" 2,0.06,1,2,1,1,0,0;
		$END;
	R(0,3.07)(11,5.82)2,0,0;
		$ATT;
		"LOWER_LEFT" (0,0) "( -30.5 meters , 20 meters )" 2,0.06,1,2,1,1,0,0;
		"UPPER_RIGHT" (0,0) "( 49.5 meters , 40 meters )" 2,0.06,1,2,1,1,0,0;
		"RELATIVE_PERMITTIVITY" (0,0) "4.26" 2,0.06,1,2,1,1,0,0;
		"LOSS_TANGENT" (0,0) "0.014" 2,0.06,1,2,1,1,0,0;
		$END;
	R(4.19375,3.07)(5.15625,3.4825)3,47,1;
		$ATT;
		"LOWER_LEFT" (0,0) "( 0 meters , 20 meters )" 2,0.06,1,2,1,1,0,0;
		"UPPER_RIGHT" (0,0) "( 7 meters , 23 meters )" 2,0.06,1,2,1,1,0,0;
		"SIGNAL_NAME" (0,0) "::Rect2R0" 2,0.06,1,2,1,1,0,0;
		"CONDUCTIVITY" (0,0) "3.0siemens/meter" 2,0.06,1,2,1,1,0,0;
		$END;
	R(5.84375,3.07)(6.80625,3.4825)3,47,1;
		$ATT;
		"LOWER_LEFT" (0,0) "( 12 meters , 20 meters )" 2,0.06,1,2,1,1,0,0;
		"UPPER_RIGHT" (0,0) "( 19 meters , 23 meters )" 2,0.06,1,2,1,1,0,0;
		"SIGNAL_NAME" (0,0) "::Rect2R1" 2,0.06,1,2,1,1,0,0;
		"CONDUCTIVITY" (0,0) "3.0siemens/meter" 2,0.06,1,2,1,1,0,0;
		$END;
$END;
$CONNECTS;
$END;
$TEXT;
$END;

