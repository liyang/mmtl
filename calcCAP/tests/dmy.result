
2002 10 10 10:46:23 zahn NMMTL_2DLF

File = /sppdg/software/src/tnt/calcCAP/tests/dmy
Number of Signal Lines  =   2
Number of Ground Planes =   1
Number of Ground Wires  =   0
Coupling Length =   0.02540 meters
Rise Time =  1000.0000 picoseconds
Contour (conductor) segments [cseg] = 10
Ground Plane/Dielectric segments [dseg] = 10
Conductivity Rect2R1 = 3 siemens/meter
Conductivity Rect2R0 = 3 siemens/meter
Note: minimum frequency for surface current assumptions is 9 MHz.
Mutual and Self Electrostatic Induction:
B(Active Signal , Passive Signal) Farads/Meter
B( ::Rect2R1 , ::Rect2R1 )=   1.2112143e-10
B( ::Rect2R1 , ::Rect2R0 )=  -6.6519686e-11
B( ::Rect2R0 , ::Rect2R1 )=  -6.6535805e-11
B( ::Rect2R0 , ::Rect2R0 )=   1.2112186e-10

Mutual and Self Inductance:
L(Active Signal , Passive Signal) Henrys/Meter
L( ::Rect2R1 , ::Rect2R1 )=   5.2066576e-07
L( ::Rect2R1 , ::Rect2R0 )=   2.7070251e-07
L( ::Rect2R0 , ::Rect2R1 )=   2.7074952e-07
L( ::Rect2R0 , ::Rect2R0 )=   5.2066366e-07

Asymmetry Ratios:

  Asymmetry ratio for inductance matrix:
     0.017366% (max), 0.017366% (average)

  Asymmetry ratio for electrostatic induction matrix:
     0.024232% (max), 0.024232% (average).


Characteristic Impedance (Ohms):
For Signal Line ::Rect2R1= 65.5645
For Signal Line ::Rect2R0= 65.5643

Characteristic Impedance Odd/Even (Ohms):
  odd= 36.4984
 even= 120.389

Effective Dielectric Constant:
For Signal Line ::Rect2R1= 4.14124
For Signal Line ::Rect2R0= 4.14124

Propagation Velocity (meters/second):
For Signal Line ::Rect2R1=   1.4731787e+08
For Signal Line ::Rect2R0=   1.4731790e+08

Propagation Velocity Odd/Even (meters/second):
  odd= 1.46015e+08
 even= 1.52127e+08

Propagation Delay (seconds/meter):
For Signal Line ::Rect2R1=   6.7880427e-09
For Signal Line ::Rect2R0=   6.7880412e-09

Propagation Delay Odd/Even (seconds/meter):
  odd= 6.8486e-09
 even= 6.57344e-09

Rdc:
Rdc(Active Signal , Passive Signal) Ohms/Meter
Rdc( ::Rect2R1 , ::Rect2R1 )=   1.5873017e-02
Rdc( ::Rect2R1 , ::Rect2R0 )=   0.0000000e+00
Rdc( ::Rect2R0 , ::Rect2R1 )=   0.0000000e+00
Rdc( ::Rect2R0 , ::Rect2R0 )=   1.5873017e-02

Far-End (Forward) Cross Talk:
FXT(Active Signal, Passive Signal)
FXT( ::Rect2R1 , ::Rect2R0 )= 2.95733e-03 =   -50.58200 dB

Near-End (Backward) Cross Talk:
BXT(Active Signal, Passive Signal)
BXT( ::Rect2R1 , ::Rect2R0 )= 9.21856e-02 =   -20.70674 dB

NOTE: Cross talk results assume there are no reflections.
