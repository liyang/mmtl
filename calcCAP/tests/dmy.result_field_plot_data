Display Units: mils
Lower Ground Plane Thickness: 1.000000e+00
Upper Extent: 4.000000e+01
Lower Extent: 0.000000e+00
Right Extent: 3.800000e+01
Left Extent: -1.900000e+01


Contour: Signal
Contour Type: Polygon
Number of Points: 4
Point: 1.200000e+01 2.000000e+01
Point: 1.200000e+01 2.300000e+01
Point: 1.900000e+01 2.300000e+01
Point: 1.900000e+01 2.000000e+01

Contour: Signal
Contour Type: Polygon
Number of Points: 4
Point: 0.000000e+00 2.000000e+01
Point: 0.000000e+00 2.300000e+01
Point: 7.000000e+00 2.300000e+01
Point: 7.000000e+00 2.000000e+01

Contour: Dielectric
Contour Type: Polygon
Number of Points: 4
Point: -1.900000e+01 2.000000e+01
Point: -1.900000e+01 4.000000e+01
Point: 3.800000e+01 4.000000e+01
Point: 3.800000e+01 2.000000e+01

Contour: Dielectric
Contour Type: Polygon
Number of Points: 4
Point: -1.900000e+01 0.000000e+00
Point: -1.900000e+01 2.000000e+01
Point: 3.800000e+01 2.000000e+01
Point: 3.800000e+01 0.000000e+00

Start Solution Output:
Active Line: Rect2R1

Element Type: Conductor
X Points: 1.200000e+01 1.287500e+01 1.375000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 0 6.666667e-01
Charge Values: 5.430602e-14 2.339074e-12 4.827496e-13

Element Type: Conductor
X Points: 1.375000e+01 1.462500e+01 1.550000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: 4.827496e-13 1.180554e-12 7.766371e-13

Element Type: Conductor
X Points: 1.550000e+01 1.637500e+01 1.725000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: 7.766371e-13 9.925373e-13 9.174748e-13

Element Type: Conductor
X Points: 1.725000e+01 1.812500e+01 1.900000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: 9.174748e-13 1.261398e-12 3.211919e-14

Element Type: Conductor
X Points: 1.900000e+01 1.900000e+01 1.900000e+01
Y Points: 2.000000e+01 2.075000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: 3.211919e-14 1.821270e-12 6.750018e-13

Element Type: Conductor
X Points: 1.900000e+01 1.900000e+01 1.900000e+01
Y Points: 2.150000e+01 2.225000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: 6.750018e-13 1.235643e-12 2.879306e-14

Element Type: Conductor
X Points: 1.900000e+01 1.812500e+01 1.725000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 0 6.666667e-01
Charge Values: 2.879306e-14 1.381576e-12 4.109419e-13

Element Type: Conductor
X Points: 1.725000e+01 1.637500e+01 1.550000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: 4.109419e-13 8.466112e-13 6.637961e-13

Element Type: Conductor
X Points: 1.550000e+01 1.462500e+01 1.375000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: 6.637961e-13 9.016962e-13 9.156214e-13

Element Type: Conductor
X Points: 1.375000e+01 1.287500e+01 1.200000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: 9.156214e-13 1.527056e-12 4.917007e-14

Element Type: Conductor
X Points: 1.200000e+01 1.200000e+01 1.200000e+01
Y Points: 2.300000e+01 2.225000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: 4.917007e-14 3.155487e-12 1.464874e-12

Element Type: Conductor
X Points: 1.200000e+01 1.200000e+01 1.200000e+01
Y Points: 2.150000e+01 2.075000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: 1.464874e-12 2.428092e-12 5.430602e-14

Element Type: Conductor
X Points: 0.000000e+00 8.750000e-01 1.750000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 0 6.666667e-01
Charge Values: -5.621650e-15 -3.164242e-13 -1.541520e-13

Element Type: Conductor
X Points: 1.750000e+00 2.625000e+00 3.500000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: -1.541520e-13 -2.857011e-13 -2.825927e-13

Element Type: Conductor
X Points: 3.500000e+00 4.375000e+00 5.250000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: -2.825927e-13 -4.434735e-13 -4.965476e-13

Element Type: Conductor
X Points: 5.250000e+00 6.125000e+00 7.000000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: -4.965476e-13 -1.028520e-12 -4.000331e-14

Element Type: Conductor
X Points: 7.000000e+00 7.000000e+00 7.000000e+00
Y Points: 2.000000e+01 2.075000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: -4.000331e-14 -2.670534e-12 -1.349871e-12

Element Type: Conductor
X Points: 7.000000e+00 7.000000e+00 7.000000e+00
Y Points: 2.150000e+01 2.225000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: -1.349871e-12 -2.086464e-12 -4.182306e-14

Element Type: Conductor
X Points: 7.000000e+00 6.125000e+00 5.250000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 0 6.666667e-01
Charge Values: -4.182306e-14 -1.666739e-12 -1.682172e-13

Element Type: Conductor
X Points: 5.250000e+00 4.375000e+00 3.500000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: -1.682172e-13 -6.667781e-13 -3.220334e-13

Element Type: Conductor
X Points: 3.500000e+00 2.625000e+00 1.750000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: -3.220334e-13 -4.237627e-13 -3.337711e-13

Element Type: Conductor
X Points: 1.750000e+00 8.750000e-01 0.000000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: -3.337711e-13 -3.917366e-13 -7.654167e-15

Element Type: Conductor
X Points: 0.000000e+00 0.000000e+00 0.000000e+00
Y Points: 2.300000e+01 2.225000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: -7.654167e-15 -3.993851e-13 -1.097482e-13

Element Type: Conductor
X Points: 0.000000e+00 0.000000e+00 0.000000e+00
Y Points: 2.150000e+01 2.075000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: -1.097482e-13 -2.283313e-13 -5.621650e-15

Element Type: Dielectric
X Points: -1.900000e+01 -1.615000e+01 -1.330000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: -2.190579e-14 -2.346450e-14 -2.486714e-14

Element Type: Dielectric
X Points: -1.330000e+01 -1.045000e+01 -7.600000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: -2.486714e-14 -2.596254e-14 -2.587525e-14

Element Type: Dielectric
X Points: -7.600000e+00 -4.750000e+00 -1.900000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: -2.587525e-14 -2.361640e-14 -1.745104e-14

Element Type: Dielectric
X Points: -1.900000e+00 9.500000e-01 3.800000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: -1.745104e-14 -5.119545e-15 1.408492e-14

Element Type: Dielectric
X Points: 3.800000e+00 6.650000e+00 9.500000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 1.408492e-14 4.064198e-14 7.025419e-14

Element Type: Dielectric
X Points: 9.500000e+00 1.235000e+01 1.520000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 7.025419e-14 9.753954e-14 1.171937e-13

Element Type: Dielectric
X Points: 1.520000e+01 1.805000e+01 2.090000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 1.171937e-13 1.247905e-13 1.217495e-13

Element Type: Dielectric
X Points: 2.090000e+01 2.375000e+01 2.660000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 1.217495e-13 1.100184e-13 9.369479e-14

Element Type: Dielectric
X Points: 2.660000e+01 2.945000e+01 3.230000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 9.369479e-14 7.622104e-14 5.935825e-14

Element Type: Dielectric
X Points: 3.230000e+01 3.515000e+01 3.800000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 5.935825e-14 4.463245e-14 3.205035e-14

End Solution Output:
Start Solution Output:
Active Line: Rect2R0

Element Type: Conductor
X Points: 1.200000e+01 1.287500e+01 1.375000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 0 6.666667e-01
Charge Values: -3.786347e-14 -1.441899e-12 -7.719984e-14

Element Type: Conductor
X Points: 1.375000e+01 1.462500e+01 1.550000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: -7.719984e-14 -5.159366e-13 -2.017711e-13

Element Type: Conductor
X Points: 1.550000e+01 1.637500e+01 1.725000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: -2.017711e-13 -2.885557e-13 -2.064045e-13

Element Type: Conductor
X Points: 1.725000e+01 1.812500e+01 1.900000e+01
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: -2.064045e-13 -2.468802e-13 -5.248591e-15

Element Type: Conductor
X Points: 1.900000e+01 1.900000e+01 1.900000e+01
Y Points: 2.000000e+01 2.075000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: -5.248591e-15 -3.119636e-13 -1.341537e-13

Element Type: Conductor
X Points: 1.900000e+01 1.900000e+01 1.900000e+01
Y Points: 2.150000e+01 2.225000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: -1.341537e-13 -2.858303e-13 -9.159332e-15

Element Type: Conductor
X Points: 1.900000e+01 1.812500e+01 1.725000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 0 6.666667e-01
Charge Values: -9.159332e-15 -5.053457e-13 -2.417678e-13

Element Type: Conductor
X Points: 1.725000e+01 1.637500e+01 1.550000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: -2.417678e-13 -4.272165e-13 -4.057875e-13

Element Type: Conductor
X Points: 1.550000e+01 1.462500e+01 1.375000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: -4.057875e-13 -5.859748e-13 -6.401037e-13

Element Type: Conductor
X Points: 1.375000e+01 1.287500e+01 1.200000e+01
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: -6.401037e-13 -1.197720e-12 -4.256708e-14

Element Type: Conductor
X Points: 1.200000e+01 1.200000e+01 1.200000e+01
Y Points: 2.300000e+01 2.225000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: -4.256708e-14 -2.782563e-12 -1.318122e-12

Element Type: Conductor
X Points: 1.200000e+01 1.200000e+01 1.200000e+01
Y Points: 2.150000e+01 2.075000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: -1.318122e-12 -2.016154e-12 -3.786347e-14

Element Type: Conductor
X Points: 0.000000e+00 8.750000e-01 1.750000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 0 6.666667e-01
Charge Values: 3.501448e-14 1.670924e-12 5.333164e-13

Element Type: Conductor
X Points: 1.750000e+00 2.625000e+00 3.500000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: 5.333164e-13 1.045324e-12 8.249907e-13

Element Type: Conductor
X Points: 3.500000e+00 4.375000e+00 5.250000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Charge Values: 8.249907e-13 1.083377e-12 1.086457e-12

Element Type: Conductor
X Points: 5.250000e+00 6.125000e+00 7.000000e+00
Y Points: 2.000000e+01 2.000000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: 1.086457e-12 1.725313e-12 5.313941e-14

Element Type: Conductor
X Points: 7.000000e+00 7.000000e+00 7.000000e+00
Y Points: 2.000000e+01 2.075000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: 5.313941e-14 3.266288e-12 1.436525e-12

Element Type: Conductor
X Points: 7.000000e+00 7.000000e+00 7.000000e+00
Y Points: 2.150000e+01 2.225000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: 1.436525e-12 2.352143e-12 4.881988e-14

Element Type: Conductor
X Points: 7.000000e+00 6.125000e+00 5.250000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 0 6.666667e-01
Charge Values: 4.881988e-14 2.087101e-12 3.615922e-13

Element Type: Conductor
X Points: 5.250000e+00 4.375000e+00 3.500000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: 3.615922e-13 9.908061e-13 6.125382e-13

Element Type: Conductor
X Points: 3.500000e+00 2.625000e+00 1.750000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Charge Values: 6.125382e-13 8.043551e-13 7.327092e-13

Element Type: Conductor
X Points: 1.750000e+00 8.750000e-01 0.000000e+00
Y Points: 2.300000e+01 2.300000e+01 2.300000e+01
Edge: 1 6.666667e-01
Charge Values: 7.327092e-13 1.038185e-12 2.767878e-14

Element Type: Conductor
X Points: 0.000000e+00 0.000000e+00 0.000000e+00
Y Points: 2.300000e+01 2.225000e+01 2.150000e+01
Edge: 0 6.666667e-01
Charge Values: 2.767878e-14 1.679999e-12 7.123114e-13

Element Type: Conductor
X Points: 0.000000e+00 0.000000e+00 0.000000e+00
Y Points: 2.150000e+01 2.075000e+01 2.000000e+01
Edge: 1 6.666667e-01
Charge Values: 7.123114e-13 1.330647e-12 3.501448e-14

Element Type: Dielectric
X Points: -1.900000e+01 -1.615000e+01 -1.330000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 3.202122e-14 4.459924e-14 5.932149e-14

Element Type: Dielectric
X Points: -1.330000e+01 -1.045000e+01 -7.600000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 5.932149e-14 7.618159e-14 9.365710e-14

Element Type: Dielectric
X Points: -7.600000e+00 -4.750000e+00 -1.900000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 9.365710e-14 1.099914e-13 1.217466e-13

Element Type: Dielectric
X Points: -1.900000e+00 9.500000e-01 3.800000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 1.217466e-13 1.248304e-13 1.172855e-13

Element Type: Dielectric
X Points: 3.800000e+00 6.650000e+00 9.500000e+00
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 1.172855e-13 9.767790e-14 7.041892e-14

Element Type: Dielectric
X Points: 9.500000e+00 1.235000e+01 1.520000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 7.041892e-14 4.080198e-14 1.421781e-14

Element Type: Dielectric
X Points: 1.520000e+01 1.805000e+01 2.090000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: 1.421781e-14 -5.024867e-15 -1.739353e-14

Element Type: Dielectric
X Points: 2.090000e+01 2.375000e+01 2.660000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: -1.739353e-14 -2.358666e-14 -2.586494e-14

Element Type: Dielectric
X Points: 2.660000e+01 2.945000e+01 3.230000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: -2.586494e-14 -2.596420e-14 -2.487589e-14

Element Type: Dielectric
X Points: 3.230000e+01 3.515000e+01 3.800000e+01
Y Points: 4.000000e+01 4.000000e+01 4.000000e+01
Charge Values: -2.487589e-14 -2.347729e-14 -2.192052e-14

End Solution Output:
