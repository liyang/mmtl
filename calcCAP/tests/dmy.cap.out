---------------------------------------
Capacitance Values
Input file : /sppdg/software/src/tnt/calcCAP/tests/dmy.cap.in
---------------------------------------
-50585.5     11700.6     
11691.5     -50585.5     
Estimated Maximum Computational Error is: -0

User CPU time: 30
 Sys CPU time: 0
